permits/models.py

```diff
@@ -1071,7 +1071,12 @@ class WorksObjectType(models.Model):
     days_before_reminder = models.IntegerField(
         _("Délai de rappel (jours)"), blank=True, null=True
     )
-
+    is_document_management_enabled = models.BooleanField(
+        _("Activer la gestion des documents"), default=False
```

Comment:
`Activer le module de gestion des documents` to be coherent with `Activer le module de publication`?